packages/treats/client/index.js

```diff
@@ -65,6 +66,21 @@ const initClient = params => {
             language
         };
 
+    if ($("#service-worker").length > 0 && "serviceWorker" in navigator) {
```

Comment:
Actually, we can use `document.querySelectorAll("script[data-treats-sw='true']")` rather than introducing jQuery as our dependency here 😄